waku/waku.md

```diff
@@ -256,15 +258,27 @@ The projection function is defined as a mapping from a 4-byte slice S to a 512-b
 	D[n] = 1
 	END FOR
 
-#### P2P Request
+A full bloom filter means that the node is to be considered a `Full Node` and it will accept any topic.
```

Comment:
Do we want to specify what a full bloom filter looks like?

Reply:
Also consistency: Bloom Filter vs Bloom filter etc - pick one, if field use `field` or similar. `Full node` should probably just be Full Node?